Kefche/Server/Web/Kefche/Controllers/ChatController.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Kefche.Controllers
+{
+    using System.Threading.Tasks;
+    using Controllers.Base;
+    using Hubs;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.SignalR;
+
+    public class ChatController : ApiController
+    {
+        private readonly IHubContext<ChatHub> hubContext;
+
+        public ChatController(IHubContext<ChatHub> hubContext)
+        {
+            this.hubContext = hubContext;
+        }
+
+        [HttpPost]
+        public async Task SendMessage(ChatMessage message) 
+            => await this.hubContext.Clients.All.SendAsync("messageReceivedFromApi", message);
```

Comment:
Are we going to use send all or is this just for testing

Reply:
Actually we are sending a message to all participants in a given chat room (although we currently have only one), right?